tests/expressions/tests.py

```diff
@@ -741,6 +741,14 @@ def test_lefthand_bitwise_and(self):
         self.assertEqual(Number.objects.get(pk=self.n.pk).integer, 40)
         self.assertEqual(Number.objects.get(pk=self.n.pk).float, Approximate(15.500, places=3))
 
+    def test_lefthand_bitwise_left_shift_operator(self):
+        Number.objects.filter(pk=self.n.pk).update(integer=F('integer').bitleftshift(2))
+        self.assertEqual(Number.objects.get(pk=self.n.pk).integer, 168)
+
+    def test_lefthand_bitwise_right_shift_operator(self):
+        Number.objects.filter(pk=self.n.pk).update(integer=F('integer').bitrightshift(2))
```

Comment:
Good idea, I added that and it didn't require changes to the implementation. I'm not certain if `FLOOR(...)` is correct but tests are passing.